enterprise/server/cmd/ci_runner/main.go

```diff
@@ -740,6 +740,9 @@ func run() error {
 
 	ws.prepareRunnerForNextInvocation(ctx, taskWorkspaceDir)
 
+	// Print an empty line to display the end time of the workflow
+	printCommandLine(ws.log, "")
```

Comment:
The blank shell prompt looks a little awkward to me - could we instead do a normal log here instead of calling `printCommandLine`, and write something like `Workflow completed in <duration>`?

Reply:
I included the timestamp instead of duration because the invocation link already includes a duration. I find the timestamp helpful because I can never remember if the duration on the invocation page includes queueing, setting up the runner etc